### DISCUSSION TITLE
Reduced dynamic Orbits (force modelling parameter)

### DISCUSSION THREAD

**Discussion by Bemtgenj** [groops-devs/groops#177] [Q&A] [answered]
Hello,

We have completed the reduced dynamic orbit calculations for 2 CubeSat missions and want to introduce a set of force modelling parameters in our next runs. 
1)	constant acceleration in the 3 LOF directions
2)	Sine/ Cosine acceleration in all directions
3)	“x” minute piece wise acceleration in all directions (constant value)
I have looked through  the GNSS processing and the documentation and I think that I will need to add parametrizationAcceleration in the leodynamicOrbits section. Then I think I would need to use accBIAS and for “1)” the constant temporal parameter and for “2)” the oscillation temporal parameter. For “3)” I would add under the constant temporal parameter a time interval of x minutes.
I wanted o check with you if this logic is correct before running the scripts as I want to make sure I understood that parametrization portion correctly and I am getting correct results with the above input

Thank you very much for you feedback and help
Cheers
Jean

**Comment by tmayerguerr** [comment, marked as answer]
Hi Jean,
Yes, you are right. Important: of course you can't estimate 1. and 3. together, because then the system of equations is no longer solvable. So either 1. or 3.
regards
Torsten

**Comment by Bemtgenj** [comment]
Hi Torsten

txs very much for the clarification, Correct - i was not planning on doing them together.

Cheers
Jean

**Reply by whulizhen** [reply]
> Hello,
> 
> We have completed the reduced dynamic orbit calculations for 2 CubeSat missions and want to introduce a set of force modelling parameters in our next runs.
> 
> 1. constant acceleration in the 3 LOF directions
> 2. Sine/ Cosine acceleration in all directions
> 3. “x” minute piece wise acceleration in all directions (constant value)
>    I have looked through  the GNSS processing and the documentation and I think that I will need to add parametrizationAcceleration in the leodynamicOrbits section. Then I think I would need to use accBIAS and for “1)” the constant temporal parameter and for “2)” the oscillation temporal parameter. For “3)” I would add under the constant temporal parameter a time interval of x minutes.
>    I wanted o check with you if this logic is correct before running the scripts as I want to make sure I understood that parametrization portion correctly and I am getting correct results with the above input
> 
> Thank you very much for you feedback and help Cheers Jean


Dear Jean
I am doing kinematic POD for CubeSat.  I cannot get the approximate position of CubeSat  as a must for the 'receiver' section in program 'GnssProcessing'. Have you got any idea of how to get the approximate position of CubeSat with Groops?

Best wishes
Zhen
